plugins/net_plugin/net_plugin.cpp

```diff
@@ -2122,7 +2122,7 @@ namespace eosio {
                            auto index = conn->pending_message_buffer.read_index();
                            conn->pending_message_buffer.peek(&message_length, sizeof(message_length), index);
                            if(message_length > def_send_buffer_size*2 || message_length == 0) {
-                              elog("incoming message length unexpected (${i})", ("i", message_length));
+                              elog("incoming message length unexpected (${i}), from ${p}", ("i", message_length)("p",boost::lexical_cast<std::string>(conn->socket->remote_endpoint())));
```

Comment:
This can throw an exception which will be caught below and report std::exception below. Pass `boost::system::error_code ec;` to `remote_endpoint(ec);` to avoid exception.